packages/l/libmysofa/xmake.lua

```diff
@@ -0,0 +1,35 @@
+package("libmysofa")
+    set_homepage("https://github.com/hoene/libmysofa")
+    set_description("Reader for AES SOFA files to get better HRTFs")
+    set_license("BSD-3-Clause")
+
+    add_urls("https://github.com/hoene/libmysofa/archive/refs/tags/$(version).tar.gz",
+             "https://github.com/hoene/libmysofa.git")
+
+    add_versions("v1.3.2", "6c5224562895977e87698a64cb7031361803d136057bba35ed4979b69ab4ba76")
+
+    add_patches("v1.3.2", "patches/v1.3.2/fix-build.patch", "5b78f981ea5fabfdc33625ede95c2b9a5edba0f570e69f063c0a59d94a551a2e")
+
+    add_deps("cmake", "zlib")
+
+    if is_plat("linux", "bsd") then
+        add_syslinks("m", "pthread")
+    end
+
+    on_install(function (package)
+        if package:is_plat("wasm", "cross") then
+            io.replace("src/CMakeLists.txt", [[find_library(MATH m)]], [[set(MATH "")]], {plain = true})
+        end
+        os.rm("windows/third-party/zlib-1.2.11")
+        os.rm("share/default.sofa")
+        os.cp("share/MIT_KEMAR_normal_pinna.sofa", "share/default.sofa")
```

Comment:
is it ```os.mv```?

Reply:
I don't know, you can check the library doc